presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/BaseElasticsearchSmokeTest.java

```diff
@@ -105,10 +104,18 @@ public void testDescribeTable()
     @Override
     public void testShowCreateTable()
     {
-        // TODO (https://github.com/prestosql/presto/issues/3385) Fix SHOW CREATE TABLE
-        assertThatThrownBy(super::testShowCreateTable)
-                .hasMessage("No PropertyMetadata for property: original-name");
-        throw new SkipException("Fix SHOW CREATE TABLE");
+        assertThat(computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .isEqualTo("CREATE TABLE elasticsearch.tpch.orders (\n" +
+                        "   clerk varchar WITH ( \"original-name\" = 'clerk', \"supports-predicates\" = false ),\n" +
+                        "   comment varchar WITH ( \"original-name\" = 'comment', \"supports-predicates\" = false ),\n" +
+                        "   custkey bigint WITH ( \"original-name\" = 'custkey', \"supports-predicates\" = true ),\n" +
+                        "   orderdate timestamp WITH ( \"original-name\" = 'orderdate', \"supports-predicates\" = true ),\n" +
+                        "   orderkey bigint WITH ( \"original-name\" = 'orderkey', \"supports-predicates\" = true ),\n" +
+                        "   orderpriority varchar WITH ( \"original-name\" = 'orderpriority', \"supports-predicates\" = false ),\n" +
+                        "   orderstatus varchar WITH ( \"original-name\" = 'orderstatus', \"supports-predicates\" = false ),\n" +
+                        "   shippriority bigint WITH ( \"original-name\" = 'shippriority', \"supports-predicates\" = true ),\n" +
+                        "   totalprice real WITH ( \"original-name\" = 'totalprice', \"supports-predicates\" = true )\n" +
```

Comment:
i am not particularily happy about these being shown here, but i didn't find a better way.

(i am actually not sure sure that SHOW queries should show hidden properties)

Reply:
i think there is a way.